rasa/core/training/story_writer/yaml_story_writer.py

```diff
@@ -91,17 +88,6 @@ def process_story_step(self, story_step: StoryStep) -> Optional[OrderedDict]:
         Returns:
             Dict with a story step.
         """
-        if self.story_contains_forms(story_step):
-            raise_warning(
-                f'Training data file contains a story "{story_step.block_name}" '
-                f"that has form(s) in it. This story cannot be converted automatically "
-                f"because of the new Rules system in Rasa Open Source "
-                f"version {LATEST_TRAINING_DATA_FORMAT_VERSION}. "
-                f"Please convert this story manually, it will be skipped now.",
-                docs=DOCS_URL_STORIES,
-            )
-            return None
```

Comment:
With this removed, is the return type of this function still `Optional`?

Reply:
No, it's not, good catch!